sos/report/plugins/kubernetes.py

```diff
@@ -296,18 +290,6 @@ def check_enabled(self):
 
     def setup(self):
         self.set_kubeconfig()
-
-        # if present, use `oc` command and add some OCP specific ressources
-        if is_executable('oc'):
```

Comment:
Removing this raises

    sos/report/plugins/kubernetes.py:18:0: W0611: Unused is_executable imported from sos.utilities (unused-import)


Please remove the unused import as well.

Reply:
My bad, done